parsing/src/recipes.ts

```diff
@@ -218,15 +218,15 @@ function getPowerGeneratingRecipes(
                 const primaryFuelPart = parts.parts[primaryFuel];
 
                 const burnDurationInMins = primaryFuelPart.energyGeneratedInMJ / burnRateMJ;
-                const burnDuration = burnDurationInMins * 60; // Convert to seconds
+                const burnDurationInS = burnDurationInMins * 60; // Convert to seconds
```

Comment:
Is this not meant to be a divide?

Reply:
This is your code. Lol, I was just adding the unit to the variable. Multiplication is right here!

Reply:
I think what's confusing me is the variable names.

`burnDurationInMins` looks like it should be `burnGenerationInS` and then what is currently `burnDurationInS` be simply `burnGenerationPerMin`.